### DISCUSSION TITLE
No sound when connecting or counting down

### DISCUSSION THREAD

**Discussion by Gittimom** [arkypita/LaserGRBL#2195] [Q&A]
I am new to using the LaserGbrl app. When I began using it, it would make a beeping sound when the laser connected and when It started and finished engraving. I went to use it today and now sounds but is still working properly. I liked to hear the beeps so I knew things were running. What did I do or need to do?

**Comment by Tribo-2dot0** [comment]
Reveja nas Configurações a aba Notification.
![Captura de tela 2024-02-06 112616](https://github.com/arkypita/LaserGRBL/assets/83982525/200567f2-e4d4-43a2-afa1-9105da99a11a)
